### ISSUE TITLE
fix: Export to PNG creates 'All Files' File

### ISSUE THREAD

**Issue by KiranBhati** [excalidraw/excalidraw#8256]
When selecting 'Export Image' choosing either PNG or SVG will only create an 'All Files' object and not a PNG / SVG

![Untitled](https://github.com/user-attachments/assets/f59908db-155a-4641-a008-6999fc4f35f8)

**Comment by AtharvSonawale** [comment]
Yeah I've seen that too, yeah there's pretty much development required.

**Comment by KiranBhati** [comment]
So is this pretty much a known issue and is being worked on, or will this sit for a while? Not a big deal either way but its always good to check!
